File: 001/prob1.py
#Project Euler Problem 1
#Add all the natural numbers below one thousand that are multiples of 3 or 5.

def isMultipleOf3or5(checkMe):
	returnValue = False
	
	if checkMe % 3 == 0 or checkMe % 5 == 0:
		returnValue = True
	
	return returnValue


i = 0
sum = 0

while i < 1000:
	if isMultipleOf3or5(i):
		sum += i
	
	i += 1

print sum

File: 001/prob1.c
//Find the sum of all the multiples of 3 or 5 below 1000.
#include <stdio.h>

int 
main() {
	int sum = 0;
	int i = 0;

	while (i < 1000)
	{
		if (i % 3 == 0 || i % 5 == 0) sum = sum + i;

		i = i + 1;
	}

	printf("The sum of all the multiples of 3 or 5 below 1000 is %d.\n", sum);

	return 0;
}
